wiki/People/The_Team/Account_support_team/id.md

```diff
@@ -25,17 +20,24 @@ Kamu harus menghubungi tim ini mengenai topik yang berhubungan dengan akun yang
 
 ### [accounts@ppy.sh](mailto:accounts@ppy.sh)
 
-- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk silence dan penghapusan konten profil.
+- [Banding atas pembatasan yang diberlakukan pada akun kamu](/wiki/Help_Centre/Account_Restrictions), termasuk:
+  - silence;
+  - penghapusan konten profil;
+  - penghapusan beatmap;
+  - penghapusan postingan dan komentar di forum.
 - Perilaku buruk yang [ingin kamu akui](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
 - [Kehilangan akses ke email](/wiki/Help_Centre#sign-in) yang terhubung dengan akun osu! kamu, atau saat akun kamu dicuri.
 - [Mengembalikan nama pengguna menjadi yang sebelumnya dan sedikit koreksi ejaan pada nama pengguna](/wiki/Help_Centre#name-changes).
 - [Pendaftaran akun bot](/wiki/Bot_Account).
 
 ### [support@ppy.sh](mailto:support@ppy.sh)
 
+*Catatan: Untuk melaporkan sebuah kasus seperti perilaku buruk dan pelecehan seksual yang ekstrim dari pengguna mana pun, termasuk dari seorang moderator, silakan kunjungi artikel [Melaporkan Penyalahgunaan](/wiki/Reporting_Bad_Behaviour/Abuse).*
+
+- Kekhawatiran terhadap [perilaku dari seorang moderator](/wiki/People/The_Team/Global_Moderation_Team#staff-complaints).
```

Comment:
```suggestion
- Kekhawatiran terhadap perilaku buruk dari seorang moderator.
```
the english ver doesn't have the hyperlink thing

Reply:
eh, didn't notice that one. thanks!